.rubocop.yml

```diff
@@ -1,2 +1,4 @@
 LineLength:
   Enabled: false
+Metrics/ParameterLists:
+  Max: 7
```

Comment:
Is this to keep rubocop from complaining about the giant list of args to the `sdm_relay_token()` function below?

Reply:
Correct.